writeengine/bulk/we_colbufcompressed.cpp

```diff
@@ -377,21 +381,31 @@ int ColumnBufferCompressed::writeToFile(int startOffset, int writeSize,
 //------------------------------------------------------------------------------
 int ColumnBufferCompressed::compressAndFlush( bool bFinishingFile )
 {
-    const int OUTPUT_BUFFER_SIZE = IDBCompressInterface::maxCompressedSize(fToBeCompressedCapacity) +
-                                   fUserPaddingBytes;
+    auto fCompressor = compress::getCompressorByType(
```

Comment:
`fCompressor` is a bit confusing, since the prefix `f` is usually used for class/struct fields. Could you please rename it (just `compressor`?) here and below?

Reply:
Thanks, fixed.